.buildkite/pipeline.yml

```diff
@@ -8,8 +8,9 @@ env:
 
 agents:
   config: cpu
```

Comment:
```suggestion

```
This line is causing the error [here](https://buildkite.com/clima/cloudy-dot-jl-ci/builds/57#018e3a32-a494-4f30-b905-b8fb0bb4813f)